actions/overlay_action.go

```diff
@@ -1,3 +1,26 @@
+/*
+Overlay Action
+
+Recursive copy of directory or file to target root filesystem.
```

Comment:
Overlay the directory or file to root filesystem by recursive copy?

Reply:
done